dipy/tracking/local/interpolation.pxd

```diff
@@ -1,7 +1,12 @@
+
 cimport numpy as np
 
-cdef int _trilinear_interpolate_c_4d(double[:, :, :, :] data, double[:] point,
-                                     double[::1] result) nogil
-cpdef trilinear_interpolate4d(double[:, :, :, :] data, double[:] point,
-                              np.ndarray out=*)
+cpdef trilinear_interpolate4d(
+    double[:, :, :, :] data,
```

Comment:
I think here the function parameters should be aligned with the `(`.

Reply:
I don't think PEP8 says anything on this subject. I can revert the [style] change but I personally find it harder to read because of the random numbers of vars on each line.

Idem for all other alignment comments.